sql_metadb/derived_tables/po_acq_unit_ids

```diff
@@ -0,0 +1,25 @@
+-- These derived table extracts aquisition unit ids from purchase order table.
+DROP TABLE IF EXISTS po_acq_unit_ids;
+
+CREATE TABLE po_acq_unit_ids AS
+    SELECT
+        po.id AS po_id,
+        po.po_number AS po_number,
+        acq_unit_ids.jsonb #>> '{}' AS po_acquisition_unit_id,
```

Comment:
Please change to data type UUID.

Reply:
from 
`acq_unit_ids.jsonb #>> '{}' AS po_acquisition_unit_id`
to
`CAST(acq_unit_ids.jsonb #>> '{}' AS uuid) AS po_acquisition_unit_id`